pinot-core/src/main/java/com/linkedin/pinot/core/query/scheduler/QueryScheduler.java

```diff
@@ -160,14 +166,17 @@ public void stop() {
     }
 
     TimerContext timerContext = queryRequest.getTimerContext();
+    int numSegmentsQueried = queryRequest.getSegmentsToQuery().size();
     LOGGER.info(
-        "Processed requestId={},table={},reqSegments={},prunedToSegmentCount={},totalExecMs={},totalTimeMs={},broker={},numDocsScanned={},scanInFilter={},scanPostFilter={},sched={}",
-        requestId, tableNameWithType, queryRequest.getSegmentsToQuery().size(),
-        queryRequest.getSegmentCountAfterPruning(), timerContext.getPhaseDurationMs(ServerQueryPhase.QUERY_PROCESSING),
+        "Processed requestId={},table={},Segments={}/{}/{},totalExecMs={},totalTimeMs={},broker={},numDocsScanned={},scanInFilter={},scanPostFilter={},sched={}",
```

Comment:
It will be good to update the log to say Segments(queried/processed/matched) - else it might be hard for someone who isn't familiar with the code to figure out what they represent; or we can log them separately.